pyDeltaRCM/command_line.py

```diff
@@ -0,0 +1,34 @@
+import os
+import argparse
+
+import pyDeltaRCM as deltaModule
+from pyDeltaRCM.deltaRCM_driver import pyDeltaRCM
```

Comment:
how about `from . import deltaRCM_driver` and then below use `deltaRCM_driver.pyDeltaRCM`. I think this is clearer as to which namespace you are using.  

It also confirms why you are probably right we should change the name of the "model" class....

Reply:
I like this solution.
>It also confirms why you are probably right we should change the name of the "model" class....

Yeah. This was what I was kindof getting at. It just means that if you want to access the module, you have to do screwy namespace stuff to avoid conflicts.